src/Components/Register.js

```diff
@@ -29,6 +34,31 @@ import {
     );
   }
 
+  registerUser() {
+    axios.post('http://localhost:8080/api/register', { 
```

Comment:
Can you format it into a manner so that we can reuse it in the future for other pages